token_server/README.md

```diff
@@ -1,4 +1,37 @@
 # Simple web server to provide EarthEngine tokens to non-whitelisted users
 
-Run with `node --experimental_modules token_server/token_server.js`.
+This is an optional component for the mapping page, that allows any user to view
+it, even if they are not whitelisted with EarthEngine. Only the site maintainer
+should follow the instructions below, and they should only need to be done once.
 
+## Deploy to Google App Engine
+
+* Follow initial steps for Google App Engine deployment: install `gcloud`; when
+prompted, log into gcloud as the GD user `gd-earthengine-user@givedirectly.org`,
+and set the gcloud project to `mapping-crisis`. Typically you can run the
```

Comment:
The bits after Typically are more specific repeats of the previous steps right? Might be nice to stick them in a legit code block

Reply:
Rewrote. I realized the reason there wasn't specific documentation was that the commands themselves are basically self-documenting: if you run a command without running a previous one, it tells you what you need to run. So got rid of this and just gave the last command to run. Now I'm not worried about documentation getting out of date.